client/src/components/Navbar.js

```diff
@@ -1,14 +1,10 @@
-import React from "react";
+import React from 'react';
 
 const Navbar = () => {
   return (
-    <header className="bg-orange fixed w-100 ph3 pv3 pv4-ns ph4-m ph5-l">
+    <header className="bg-orange w-100 ph3 pv3 pv4-ns ph4-m ph5-l">
```

Comment:
instead of `ph3` and `pv3` you could do just `pa3` since `ph` is padding top and bottom and `pv` is padding left and right. and `pa` is all of those